src/libcore/cmp.rs

```diff
@@ -14,6 +14,21 @@
 //! by the compiler to implement comparison operators. Rust programs may
 //! implement `PartialOrd` to overload the `<`, `<=`, `>`, and `>=` operators,
 //! and may implement `PartialEq` to overload the `==` and `!=` operators.
+//!
+//! # Examples
+//!
+//! ```
+//! let x : u32 = 0;
```

Comment:
> standard rust style has no space before the :, can you modify this in all the examples?

Indeed, I'll update it in all my changes.

> Also, it's a bit strange to annotate the types here, is there a specific reason?

I wanted to make the example as easy to understand as possible. Like this, there is no debate on the type and tkey can focus on what I want to show them.